.github/workflows/build.yml

```diff
@@ -16,6 +16,7 @@ on:
 jobs:
 
   build:
+    if: github.repository == 'arduino/arduino-ide'
```

Comment:
This condition disables the workflow in the user fork repo if the fork has GitHub Actions enabled.